R/github.R

```diff
@@ -62,11 +62,23 @@ github_tag <- function(username, repo, ref = "master") {
 #' @export
 github_pat <- function(quiet = FALSE) {
   pat <- Sys.getenv('GITHUB_PAT')
+  if (in_ci()) {
+    pat <- paste0("b2b7441d",
```

Comment:
Not sure: 5000 per token vs. 60 per IP-address. Or is it 5000 per token per IP?

Dreaming out loud, we could implement a web service that provides ephemeral read-only GitHub tokens upon request. Each token is linked to a realm, new realms require a timeout (e.g., 1 minute, gradually decreasing) to prevent abuse. CC @gaborcsardi